lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1454,10 +1456,8 @@ def config_ipv6_iface(self, cmd):
 
         if ret_code == 0:
             print("Configuration successful")
-        elif ret_code == 1:
-            print("Configuration not changed")
         else:
-            print("Failed to configure")
+            raise Exception("Failed to configure the IPV6 interface!")
```

Comment:
This is to make the test fail early?

Reply:
to make it fail at all - the test was not failing if the script was not executed successfully

Reply:
See also https://github.com/magma/magma/pull/13982 why it would be good for this step to fail if the script has problems - it was only visible by info logging that the script was not executed.